docs/application-connector/10-03-application-registry-troubleshooting.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Application Registry Troubleshooting
```

Comment:
As a user, I don't really know the architecture. I wouldn't know that the error I got is because of an App. Registry. mishap.

Reply:
Instead, I'd try to come up with a name that stresses the fact that these are certificate-related errors.